diff -Naur a/sql/startsql.cpp b/sql/startsql.cpp
--- a/sql/startsql.cpp	2025-06-14 12:51:03.000000000 +0200
+++ b/sql/startsql.cpp	2026-08-03 05:22:56.411860244 +0200
@@ -325,7 +325,7 @@
     }
     // Write every line of lesson content to database
     for (int i = 0; i < content.size(); i++) {
-        // simplifiedContent = QString::QString(
+        // simplifiedContent = QString(
         //	content.at(i)).replace(QChar(0x27), "''",
         // Qt::CaseSensitive).simplified();
         simplifiedContent = QString(content.at(i))
diff -Naur a/widget/lessondialog.cpp b/widget/lessondialog.cpp
--- a/widget/lessondialog.cpp	2025-06-14 12:51:03.000000000 +0200
+++ b/widget/lessondialog.cpp	2026-08-03 05:22:56.411918675 +0200
@@ -72,9 +72,7 @@
                 "\n", Qt::SkipEmptyParts);
             // Delete empty lines
             for (int i = 0; i < lessonData->size(); i++) {
-                if (QString(lessonData->at(i).toLocal8Bit().constData())
-                        .simplified()
-                    == "") {
+                if (lessonData->at(i).simplified().isEmpty()) {
                     lessonData->removeAt(i);
                 }
             }
@@ -238,8 +236,7 @@
         = lineLessonContent->toPlainText().split("\n", Qt::SkipEmptyParts);
     // Delete empty lines
     for (int i = 0; i < contentList.size(); i++) {
-        if (QString(contentList.at(i).toLocal8Bit().constData()).simplified()
-            == "") {
+        if (contentList.at(i).simplified().isEmpty()) {
             contentList.removeAt(i);
         }
     }
